method-proposals/PROPOSAL-did-cheqd.md

```diff
@@ -0,0 +1,142 @@
+# DID method standardization proposal: did:cheqd
+
+This is a proposal to include `did:cheqd` in the initial set of DID methods that will be standardized by the DID Methods WG.
+
+## Description
+
+`did:cheqd` is a blockchain-based DID Method that runs atop the cheqd Network. 
+
+- Summary of [DID Method Traits](https://identity.foundation/did-traits/#comparison-of-did-methods)
+
+### Benefits
+
+* Immutable, cryptographically verifiable history of DID Documents and DID Document updates
+* High assurance in integrity of DID Documents and DID Document history, based on delegated Proof of Stake (dPoS) consensus algorithm
+* Key rotation supported with fully indexed, chronologically ordered and searchable history of DID Documents and associated updates
+* Hosted in a decentralized way, eliminating reliance on DNS, web servers or other centralized storage
```

Comment:
Not sure I'd say this is true, since any system does rely on DNS, web servers etc but I guess the point you're trying to make is that it's got HA / not a set of centralised servers.

Reply:
Removed as it may have been misconstrued